recipes/app3.rb

```diff
@@ -138,8 +138,8 @@
 etherpad_tag = '1.8.6-2020.11.13.2015'
 etherpad_snowdrift_tag = '1.8.6-2020.11.13.2015'
 
-etherpad_creds = data_bag_item('etherpad', 'mysql_creds_osl')
-etherpad_snowdrift_creds = data_bag_item('etherpad', 'mysql_creds_snowdrift')
+etherpad_creds = data_bag_item('etherpad', 'creds_osl')
+etherpad_snowdrift_creds = data_bag_item('etherpad', 'creds_snowdrift')
```

Comment:
I think this might be a better naming scheme for these.
```suggestion
etherpad_secrets_osl = data_bag_item('etherpad', 'osl')
etherpad_secrets_snowdrift = data_bag_item('etherpad', 'snowdrift')
```